### ISSUE TITLE
split history in multi page

### ISSUE THREAD

**Issue by Bananeweizen** [cgeo/cgeo#4672]
I'm thinking of making the history list a multi page (like cache details), where we can more easily show different history lists side by side
* recent caches (listing what you also see in the search field of the main activity)
* field notes (listing only caches with offline logs)
* recently logged (this is our current history)

What do you think about that?

**Comment by Bananeweizen** [comment]
And I just had a really nice additional idea. 4th page: recently deleted caches (i.e. the ones which are still in the database because of 3 days not over yet)

**Comment by SammysHP** [comment]
Good idea. Maybe it also helps with the issue about the understanding of our history (caches logged in c:geo vs. logged via browser and/or opened in c:geo).

**Comment by Lineflyer** [comment]
+1  This sounds all nice (incl. the 4th page).

**Comment by MrYaz** [comment]
+1 This sounds like a really good idea!
It would solve issue #4334 as well!

Go for it!
